torcheval/metrics/toolkit.py

```diff
@@ -11,11 +11,11 @@
 import torch
 import torch.distributed as dist
 
-from torcheval.metrics import Metric
```

Comment:
The file reorder is different from internal meta lint -- since here usort treated torcheval as a first-party library where internal usort will ignore this first-party detection.

Let's add a project.toml and ignore it to match the 2 behaviours

https://github.com/facebookresearch/recipes/blob/bc3c7e9424a8ab6444917b23098061abbb3205b7/pyproject.toml#L1-L2